.pre-commit-config.yaml

```diff
@@ -1,37 +1,38 @@
 repos:
   - repo: https://github.com/python-poetry/poetry
-    rev: "fb493f2e80a11403fa289617a703ae622d4ccd75" # frozen: 1.3.0
+    rev: "7fe4678f92dcab48a35169db6567cbb3f5a7e3ef" # frozen: 1.4.0
```

Comment:
the latest version is `1.4.2`, shouldn't we use the latest version?

Reply:
Weird, that's the output of `pre-commit autoupdate`.